db/system_keyspace.cc

```diff
@@ -306,6 +307,7 @@ schema_ptr system_keyspace::built_indexes() {
        );
        builder.set_gc_grace_seconds(0);
        builder.with_version(generate_schema_version(builder.uuid()));
+       builder.set_wait_for_sync_to_commitlog(true);
```

Comment:
@psarna please comment on this. is it critical, or can we tolerate missed writes? 

I guess it is low intensity so we can err on the side of being cautious.